beacon_node/network/src/service.rs

```diff
@@ -544,6 +546,7 @@ fn spawn_service<T: BeaconChainTypes>(
                                 }
                             }
                             BehaviourEvent::PeerSubscribed(_, _) => {},
+                            BehaviourEvent::PeerUnsubscribed(_, _) => {},
```

Comment:
Maybe its better to remove both these events entirely. Will increase performance a little bit and we can add them back when/if we need them.